exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -454,4 +437,11 @@
   .perseus-hint-label:after
     content: ')'
 
+  .paragraph
+    padding: 4px
+
+  .svg-image *
+    padding-bottom: 0 !important
+
+
```

Comment:
I think un-scoped css only apply down the tree. 
Since we don't directly modify perseus, this un-scoped css and `!important` are used to fight its original styling.